src/lj_jit.h

```diff
@@ -166,7 +166,11 @@ typedef struct SnapShot {
 #define SNAPCOUNT_DONE	255	/* Already compiled and linked a side trace. */
 
 /* Compressed snapshot entry. */
+#ifdef LJ_GC64
+typedef uint64_t SnapEntry;
```

Comment:
This is one way of making snapshot entries work, though I suspect that it would be more efficient to instead keep them as 32-bit values, and use pairs of them for frame links.